db_config/test/pg_hba.conf

```diff
@@ -1,4 +1,5 @@
 # Allow any user on the local system to connect
 # TYPE  DATABASE        USER            ADDRESS                             METHOD
 local   all             all                                                 trust
-hostssl all             all             all                                 md5
+host    all             all             all                                 md5
+hostssl all             all             all                                 md5
```

Comment:
As discussed async, this change shouldn't be necessary and it's not related to the scope of the PR IMO. 🙂